cmd/oauth/oauth.go

```diff
@@ -89,8 +89,20 @@ func main() {
 		os.Exit(1)
 	}
 
-	strg, err := cmd.InitTokenStorage(ctx, &args.CommonCliArgs)
+	storage, err := cmd.CreateInitializedSecretStorage(ctx, &args.CommonCliArgs)
 	if err != nil {
+		setupLog.Error(err, "failed to construct the secret storage")
+		os.Exit(1)
+	}
+	if err := storage.Initialize(ctx); err != nil {
+		setupLog.Error(err, "failed to initialize the secret storage")
+		os.Exit(1)
+	}
```

Comment:
Is the `Initialized` in CreateInitializedSecretStorage() meant differently than the `storage.Initialize()`? Because I would assume that the storage created by CreateInitializedSecretStorage does not need to be further initialized.

Reply:
🫨

Reply:
God, I added this on two places and forgot to remove them twice 🙄